liquibase-standard/src/main/java/liquibase/changelog/DatabaseChangeLog.java

```diff
@@ -924,11 +924,12 @@ public boolean include(String fileName,
         if (isRelativePath) {
             try {
                 fileName = resourceAccessor.get(this.getPhysicalFilePath()).resolveSibling(fileName).getPath();
-                fileName = normalizePath(Paths.get(fileName).toString());
             } catch (IOException e) {
                 throw new UnexpectedLiquibaseException(e);
             }
         }
+        final String normalizedFilePath = normalizePath(Paths.get(fileName).toString());
```

Comment:
Is there a reason you moved this from its original location?

Reply:
had to make it final to pass it to the runner

Reply:
As long as moving it outside of the `if(isrelativePath)` block is okay, then I am fine with this.